remoting/getty/getty_client.go

```diff
@@ -116,6 +117,7 @@ type Client struct {
 	addr           string
 	opts           Options
 	conf           ClientConfig
+	mux            sync.RWMutex
```

Comment:
Why don't you use anonymous combination in this case?

Reply:
may be will add GetMux function to get lock of client

Reply:
- -